test_conformance/api/test_api_min_max.cpp

```diff
@@ -614,14 +616,16 @@ int test_min_max_mem_alloc_size(cl_device_id deviceID, cl_context context,
     {
         log_error(
             "ERROR: Reported max allocation size is less than required of "
-            "total memory! (%llu or %lluMB, from a total mem size of %lluMB)\n",
+            "total memory! (%" PRIu64 " or %" PRIu64
+            "MB, from a total mem size of %" PRIu64 "MB)\n",
             maxAllocSize, (maxAllocSize / 1024) / 1024,
             (requiredAllocSize / 1024) / 1024);
         return -1;
     }
 
-    log_info("Reported max allocation size of %lld bytes (%gMB) and global mem "
-             "size of %lld bytes (%gMB).\n",
+    log_info("Reported max allocation size of %" PRId64
```

Comment:
We're using a `PRId64` to print `cl_ulong` values, i'd probably prefer `PRIu64` to match the sign.

Reply:
Fully agree, I think I've fixed all occurrences with the latest push.